core/src/main/java/nl/nn/adapterframework/pipes/SenderPipe.java

```diff
@@ -34,27 +35,32 @@
 @Category("Basic")
 public class SenderPipe extends MessageSendingPipe {
 
-	@Override // to obtain proper order in FrankDoc and XSD
```

Comment:
Het is wel handig ergens in deze klasse een dergelijke tekst te zetten waarom deze methodes hier op deze manier in staan.

Reply:
In my opinion, the @Reintroduce annotation should make this clear already. I have updated the JavaDoc comment of the Reintroduce class. If you hover over the @Reintroduce annotation, you see the explanation.